specification/cosmos-db/resource-manager/Microsoft.DocumentDB/preview/2021-04-01-preview/cosmos-db.json

```diff
@@ -5202,6 +5202,282 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DocumentDB/databaseAccounts/{accountName}/services/{serviceName}": {
```

Comment:
note: we might need to add /default at the end of this URL since for each service type a singleton instance of resource will exist. User cannot create multiple DataTransfer resources. 

Similar to what is done in throughputSettingsResource